.gitignore

```diff
@@ -1 +1,2 @@
-/.vscode
\ No newline at end of file
+/.vscode
+vitess-framework-testing/
```

Comment:
Out of curiosity, what is this required for?

Reply:
This change seems to have been from @Akilan1999, he'll know more about it.

Reply:
It was a local change that accidentally got pushed. Removed in the last commit.